### PR TITLE
model encryption and decryption

### PR DESCRIPTION
**Pull request by chenwhql**

### CODE REVIEW

File: cmake/external/wbaes.cmake [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+SET(WBAES_INSTALL_DIR   ${WBAES_INSTALL_ROOT}/${WBAES_DST_DIR})
+SET(WBAES_ROOT          ${WBAES_INSTALL_DIR})
+SET(WBAES_INC_DIR       ${WBAES_ROOT}/include)
+SET(WBAES_LIB_DIR       ${WBAES_ROOT}/lib)
+
+SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_RPATH}" "${WBAES_ROOT}/lib")
+SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+
+SET(WBAES_TAG   "v1.0.0" CACHE STRING "" FORCE)
+SET(WBAES_URL   "https://paddlepaddledeps.bj.bcebos.com/wbaes-sdk.linux-x86_64.${WBAES_TAG}.tgz" CACHE STRING "" FORCE)
```
Comment (luotao1) [inline cmake/external/wbaes.cmake]: bj->cdn

File: cmake/external/wbaes.cmake [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+IF(APPLE)
+    MESSAGE(WARNING "Mac is not supported with WBAES in Paddle yet.")
+    return()
+ENDIF()
+
+IF(WIN32)
+    MESSAGE(WARNING "Win32 is not supported with WBAES in Paddle yet.")
+    return()
+ENDIF()
```
Comment (luotao1) [inline cmake/external/wbaes.cmake:18]: 15-23行可以写在一块
```
IF(APPLE or WIN32)
   MESSAGE(WARNING "APPLE and Win32 are not supported with WBAES in Paddle yet.")
    return()
endif()
```

File: cmake/external/wbaes.cmake (hunk 1 above)
Comment (chenwhql) [reply cmake/external/wbaes.cmake]: Down

File: cmake/external/wbaes.cmake (hunk 2 above)
Comment (chenwhql) [reply cmake/external/wbaes.cmake:18]: Down

**Pull request closed without merging**